recipes/pynwb/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pynwb" %}
+{% set version = "0.1" %}
+{% set org = "NeurodataWithoutBorders" %}
+{% set branch = "dev" %}
+{% set sha256 = "e97fd13b714303ea1fc4bff1cec52a3d2eaa0ad33f45265651774af8409010fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:  https://github.com/{{ org }}/{{ name }}/archive/{{ branch }}.tar.gz
```

Comment:
Does `branch == dev` guarantee the correct version? It seems potentially fragile.  It seems safer to maybe use the pypi tarball:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18